gatsby_fp/src/components/Layout/Body/Fields/GenInfoTable.js

```diff
@@ -46,6 +47,18 @@ export default function GenInfoTable({
             />
           </div>
         )}
+        {(!fluidData && publicURL) && (
```

Comment:
Is it possible for these to have more descriptive names, like have "image" or "photo" in the name? They're a bit ambiguous otherwise.

Reply:
Sure, I will rename them to imageFluidData and imagePublicURL.